pkg/power/platform/source/redfish_util.go

```diff
@@ -63,19 +74,33 @@ func getRedfishModel(access RedfishAccessInfo, endpoint string, model interface{
 		return fmt.Errorf("server returned status: %v", resp.Status)
 	}
 
+	dec := json.NewDecoder(resp.Body)
+	dec.DisallowUnknownFields()
+
+	var returnErr error
 	// Decode the response body into the provided model struct
-	err = json.NewDecoder(resp.Body).Decode(model)
-	if err != nil {
-		return err
+	decErr := dec.Decode(model)
+
+	// process the error and only return significant ones
+	if decErr != nil {
+		if strings.HasPrefix(decErr.Error(), "json: unknown field ") {
+			// ignore unknown field error
+			fieldName := strings.TrimPrefix(decErr.Error(), "json: unknown field ")
```

Comment:
Dell returns some fields that are not defined in the power model, ignore them here